presto-main/src/main/java/io/prestosql/operator/OperatorStats.java

```diff
@@ -457,6 +469,8 @@ public OperatorStats add(Iterable<OperatorStats> operators)
 
         Optional<BlockedReason> blockedReason = this.blockedReason;
 
+        long dynamicFilterSplitsProcessed = this.dynamicFilterSplitsProcessed;
```

Comment:
move this above `long physicalWrittenDataSize = this.physicalWrittenDataSize.toBytes();`